backend/src/main/java/com/znane_groby/backend/BackendApplication.java

```diff
@@ -0,0 +1,21 @@
+package com.znane_groby.backend;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class BackendApplication {
+    public static void main(String[] args) {
+        SpringApplication.run(BackendApplication.class, args);
+    }
+
+    private static final Logger log = LoggerFactory.getLogger(BackendApplication.class);
```

Comment:
Lepiej dodać do klasy adnotację lomboka '@Slf4j', mniej śmieci w kodzie a robi dokładnie to samo, zob. https://projectlombok.org/features/log

Reply:
Oki (było wzięte, z jakiegoś tutoriala, ale zgadzam się, że @slf4j jest fajniejsze i w sumie nie zachowałem spójności)